app/models/wrangling_guideline.rb

```diff
@@ -0,0 +1,14 @@
+class WranglingGuideline < ActiveRecord::Base
+  acts_as_list
+
+  attr_protected :content_sanitizer_version
+
+  validates_presence_of :content, :title
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.